### PR TITLE
D-Bus API docs inside introspection XML, part2

### PR DESCRIPTION
**Pull request by mvidner**
## Problem

- https://trello.com/c/LN220ePd/23-document-d-bus-api-in-sync-with-code

## Solution

- continuation of #415
- fixed some markup, with a better understanding how the format works

migrating these interfaces:
<ul>
<li>org.opensuse.Agama.Questions1</li>
<li>org.opensuse.Agama.Users1</li>
<li>org.opensuse.Agama1.Progress</li>
<li>org.opensuse.Agama1.ServiceStatus</li>
<li>org.opensuse.Agama1.Validation</li>
</ul>

Still TODO, for a next PR: the  **Storage** gorilla

### CODE REVIEW

File: doc/dbus/index.html [hunk 1]
```diff
@@ -1,6 +1,26 @@
+<!--
+echo "<ul>" ;\
```
Comment (lslezak) [inline doc/dbus/index.html:2]: Um, so could we directly generate it from the `Makefile`?

**Review by lslezak — APPROVED** [review]
LGTM

File: doc/dbus/index.html (hunk 1 above)
Comment (mvidner) [reply doc/dbus/index.html:2]: Quite likely :smile: , in the next PR

**Comment by mvidner** [comment]
Hmm the deployment failed. I should improve the PR CI

**Pull request merged**